include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -212,4 +212,85 @@ def YieldOp : VerifOp<"yield", [
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// Verification intent ops
+//===----------------------------------------------------------------------===//
+
+/// BMC -- short for bounded model check.
+def BMC : I32EnumAttrCase<"BMC", 0, "bmc">;
+/// KIP -- short for k-Inductive-Proof.
+def KIP : I32EnumAttrCase<"KIP", 0, "induction">;
```

Comment:
Maybe spell out the attribute names, e.g., `KInductiveProof` instead of `KIP`. Otherwise, people will probably have to frequently look up the meaning in the documentation here.

Reply:
Good point, I'll update that!